src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs

```diff
@@ -21,7 +21,7 @@ private enum LanguageGenerals
             [Display("ja-JP")] Japanese,
             [Display("en-US")] Jabber,
             [Display("ko-KR")] Korean,
-            [Display("zh-CN")] ChineseSimplified,
+            [Display("zh-HK")] ChineseTraditional,
```

Comment:
Are you sure that the other one isn't used at all / can be replaced?

Reply:
Even if the other one (simplified Chinese) exist, we don't know it's language code in Generals, so... for now I think it doesn't have a place in this enum, so it should just be replaced

Reply:
Alright